purchases/src/main/kotlin/com/revenuecat/purchases/common/responses/PriceResponse.kt

```diff
@@ -0,0 +1,30 @@
+@file:JvmSynthetic
+
+package com.revenuecat.purchases.common.responses
+
+import com.revenuecat.purchases.models.Price
+import com.revenuecat.purchases.utils.MICRO_MULTIPLIER
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.text.NumberFormat
+import java.util.Currency
+import java.util.Locale
+
+@Serializable
+internal data class PriceResponse(
```

Comment:
Hmm I wondered if it would make sense to make this a nested class inside the `SubscriptionInfoResponse` to avoid increasing the top level API surface... But I think it's fine as this as well.